chain/x/zoracle/client/rest/rest.go

```diff
@@ -14,7 +14,8 @@ const (
 )
 
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string) {
-	r.HandleFunc(fmt.Sprintf("/%s/request/{%s}", storeName, requestID), getRequestHandler(cliCtx, storeName)).Methods("GET")
+	r.HandleFunc(fmt.Sprintf("/%s/request/{%s}", storeName, requestID), getRequestByIDHandler(cliCtx, storeName)).Methods("GET")
```

Comment:
Can you rename variable to `requestIDTag`? Having variable name `requestID` on top scope is confusing.